packages/fxa-auth-client/package.json

```diff
@@ -2,20 +2,36 @@
   "name": "fxa-auth-client",
   "version": "0.0.0",
   "description": "",
-  "main": "dist/server/packages/fxa-auth-client/server.js",
+  "main": "dist/server/cjs/packages/fxa-auth-client/server.js",
+  "module": "dist/server/esm/packages/fxa-auth-client/server.js",
   "exports": {
-    ".": "./dist/server/packages/fxa-auth-client/server.js",
+    ".": {
```

Comment:
Previously it didn't matter that fxa-auth-client was transpiled into cjs since Webpack 4 was not using "exports" and loading the original source directly for Babel to transpile.  With Webpack 5's support for "exports", it became a problem.  Settings' `import` of something from this package did not work.  A dual format package allows Settings' `import` to work while maintaining compatibility with whatever required this package to be cjs in the first place.

Reply:
> I was going to suggest possibly filing a follow up for investigating this or `fxa-shared`'s need for this sometime down the road but that'd probably be another whack-a-mole scenario with unknown amount of refactoring

By "investigating this" do you mean not needing to export both formats?

> Plus would be nice not to need `fxa-auth-client` in the future. 

What's going to replace it in gql-api?

> Not sure if `fxa-auth-client`'s (non-existent??) README / `fxa-shared`'s README is the best place for a note but could be useful as a note or comment somewhere. 

Couldn't hurt I guess.  But I suspect people open a package's package.json more often than the README.

Reply:
> By "investigating this" do you mean not needing to export both formats?

Yeah, would be nice one day, heh.

> > Plus would be nice not to need fxa-auth-client in the future.
>
> What's going to replace it in gql-api?

I saved this note from Ben a while back (have it in my WIP Settings architecture doc so it's not lost):

Until we have an abstracted set of business logic outside the auth-server routing code, we need to call into auth-server for most of the functions (which auth-client does). But essentially, removing the use of the auth-client in graphql-api server needs the following steps done for each use of it:
* Analyze the URL the auth-client calls to determine all db calls and logic that the route handles
* Ensure the db logic is done in db code, refactor into fxa-shared as needed
* Ensure all needed business logic for the route is handled in fxa-shared, ideally via a new FirefoxAccount class
* Switch auth-client call to appropriate business logic calls into fxa-shared

> Couldn't hurt I guess. But I suspect people open a package's package.json more often than the README.

Yeah maybe that's true, I'll leave it up to you🤷‍♀️